packages/parang/src/helper/storage.js

```diff
@@ -0,0 +1,23 @@
+const USER_TOKEN = 'USER_TOKEN';
+
+export const setUserToken = token => {
+  try {
+    if (!token) throw new Error('유효하지 않은 토큰입니다.');
+  } catch (error) {
+    console.error(error);
+    alert(error.message);
+  }
```

Comment:
```suggestion
if (!token) return alert('유효하지 않은 토큰입니다.');
```

여기서는 return alert이 심플해 보이는것 같은데 try를 사용하신 다른 이유가 있는건지 궁금해요

Reply:
맞아요! 이건 씰 말대로 반영하는게 좋겠네요😄